src/components/map/aboveMap/TravelTimeBox.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import { useViewsStore } from '@/stores/views'
+import { viewList } from '@/model/views.model'
+import Clock from '@/assets/icons/clock.svg'
+
+const props = defineProps<{
+  travelTime: number
+  topPosition: number
+  leftPosition: number
+  isHighlighted: boolean
+}>()
+
+const viewStore = useViewsStore()
+
+const positionStyle = computed(() => {
+  let style: string = ''
+  const size_text = 18
```

Comment:
You can increase the size_text const, because some time the size of window grow 
![image](https://user-images.githubusercontent.com/121813008/221520349-9897dd8c-e1c8-459f-a607-eba53ad5bf72.png)